app/assets/javascripts/organizations.js

```diff
@@ -0,0 +1,65 @@
+$(function () {
+
+  "use strict";
+
+  $(document).ready(function () {
+    $(".btn.new_account").on("ajax:success", function (e, data) {
+      $(".new_account_form").empty();
+      $(".previous_buttons").addClass("hidden");
+      $(".next_step_wrapper").addClass("hidden");
+      $(".new_account_form").append(data);
+      $("#new_bank_account").on("ajax:success", function (e, data) {
+        if (data["status"] === "error") {
```

Comment:
@dissident Not sure why do event need to bind anything to `ajax:success` but you always may do it with
`$(document).on("ajax:success", "#new_bank_account", function(...){...})`. Read jquery `on` documentation